clients/web/src/views/layout/LoginView.js

```diff
@@ -54,10 +54,6 @@ var LoginView = View.extend({
             }, this));
         },
 
-        'click a.g-register-link': function () {
-            events.trigger('g:registerUi');
```

Comment:
I'm concerned about what the loss if these event triggers will do to the reusability of the `LoginView`, etc. While triggering events on the global event bus (the current behavior) isn't ideal, it does not impose any particular routing setup on the owner of the view. Changing this will force any user of the `LoginView` to adapt the same user-facing routing scheme as Girder's web client app.

Accordingly, I think we should skip adding `href` for any behavior that is implemented by an `events.trigger` (in all the places in this PR).

Getting rid of `router.navigate` is a worthwhile change, and doesn't have any broader implications for reusability.

Reply:
Indeed, this would break how we use the login dialog in HistomicsTK.  I think you could get away with leaving in the href if you added a `preventDefault` call in these handlers.

Reply:
I put back the events for the login/register/forgot-password dialogs. Let me know if it looks ok.